nuxeo-dm/nuxeo-platform-webapp-core/src/main/java/org/nuxeo/ecm/webapp/action/ConversionActionBean.java

```diff
@@ -53,6 +54,8 @@
 import org.nuxeo.ecm.core.convert.api.ConverterNotRegistered;
 import static org.nuxeo.ecm.platform.mimetype.interfaces.MimetypeRegistry.PDF_MIMETYPE;
 import static org.nuxeo.ecm.platform.mimetype.interfaces.MimetypeRegistry.PDF_EXTENSION;
+
```

Comment:
static import should be before all others imports. I know it's not your changes, but we lead to something weird now with a blank line between two nuxeo imports.